tests/CMakeLists.txt

```diff
@@ -28,6 +28,10 @@ if(MSVC)
 
 	# Add linker flags
 	set(CMAKE_EXE_LINKER_FLAGS /DEBUG)
+else()
+	if(CMAKE_SYSTEM_PROCESSOR MATCHES "x86" OR CMAKE_SYSTEM_PROCESSOR MATCHES "amd64")
+		target_compile_options(acl_unit_tests PRIVATE "-msse4.1")
```

Comment:
avx is almost 10yo but sse 4.1 is probably safer. It'd really be nice if we could handle this in a nicer way, I have a trello card for this somewhere and ended up hacking it with python and make.py for now for AVX support.